specs/core/0_beacon-chain.md

```diff
@@ -394,6 +397,11 @@ A `SpecialRecord` object has the following fields:
     'randao_commitment': 'hash32',
     # Slots the proposer has skipped (ie. layers of RANDAO expected)
     'randao_skips': 'uint64',
+    # Proof of custody commitment
+    'proof_of_custody_commitment': 'hash32',
+    # Slot the proof of custody seed was last changed
+    'proof_of_custody_last_change': 'uint64',
```

Comment:
Can this be `latest_proof_of_custody_change_slot` to match other latest X change slot (like `last_status_change_slot` in this defintion).